packages/xo-web/src/common/intl/messages.js

```diff
@@ -1520,7 +1520,8 @@ const messages = {
   deleteDefaultTemplatesTitle: 'Delete default template{nDefaultTemplates, plural, one {} other {s}}',
   deleteDefaultTemplatesMessage:
     'You are attempting to delete {nDefaultTemplates, number} default template{nDefaultTemplates, plural, one {} other {s}}. Do you want to continue?',
-
+  deleteProtectedTemplatesTitle:'Delete protected template{nProtectedTemplates, plural, one {} other {s}}',
+  deleteProtectedTemplatesMessage: 'You are attempting to delete {nProtectedTemplates, number} protected template{nProtectedTemplates, plural, one {} other {s}}. Do you want to continue?',
```

Comment:
I'm not a fan of the `protected template` naming.
Do you think the term `protected template` is clear enough for users to understand `protected against accidental deletion`?

Reply:
I agree with @MathieuRA, let's be as explicit as possible with "protected against accidental deletion" since that's the label we used for this option and users might not know what protected or secured means for a template, it could mean many different things.